nextcloud_users

```diff
@@ -51,6 +56,11 @@ class NextcloudUsers:
 		# append for every key in users the key and the value to the results
 		[data['nextcloud_users'].append(str(key) + ".value " + str(users[key]))
 			for key in users.keys()]
+                
+		#storage
+		data['nextcloud_users'].append('num_users.value %s' % num_users)
```

Comment:
`num_users` is used before it is assigned. I think you may have copied the append statement from 2 lines down mistakenly.